roles/config-rh-sso/tasks/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+# tasks file for Red Hat SSO
+
+- import_tasks: 'prereq.yml'
+- import_tasks: 'setup-rh-sso.yml'
+- import_tasks: 'configure-rh-sso.yml'
+- include_tasks: 'realm.yml'
```

Comment:
Can we move the call to `realm.yml` into the `configure-rh-sso` file? In that way it makes it easier to create a playbook that's meant for *just* configuration.

Reply:
Yes, absolutely. That makes a lot of sense. I have put the configuration specific parts inside the `configure-rh-sso` as suggested. Thank you !